pkg/infrastructure/clusterapi/clusterapi.go

```diff
@@ -310,6 +301,14 @@ func (i *InfraProvider) Provision(dir string, parents asset.Parents) ([]*asset.F
 	}
 
 	logrus.Infof("Cluster API resources have been created. Waiting for cluster to become ready...")
+
```

Comment:
Did this sneak into this PR? This doesn't seem directly related to the work on contexts.